src/TBuffer.cpp

```diff
@@ -4272,8 +4300,8 @@ bool TBuffer::processGBSequence(const std::string& bufferData, const bool isFrom
                 [[clang::fallthrough]];
             case 1:
 #if defined(DEBUG_GB_PROCESSING)
-                qDebug().nospace() << "TBuffer::processGBSequence(...) " << gbSequenceLength << "-byte " << (isGB18030 ? "GB18030" : "GB2312/GBK") << " sequence accepted, it was " << codePoint.size()
-                                   << " QChar(s) long [" << codePoint << "]";
+                qDebug().nospace() << "TBuffer::processGBSequence(...) " << gbSequenceLength << "-byte " << (isGB18030 ? "GB18030" : "GB2312/GBK") << " sequence accepted, it is " << codePoint.size()
+                                   << " QChar(s) long [" << codePoint << "] and is in the " << dataIdentity.c_str() << " range";
```

Comment:
I was assigning a value to `dataIdentity` for most (now all) of the ranges of valid characters but I was not displaying it...